Tests/AutoTests/PerformanceTest/LoadTesting.js

```diff
@@ -0,0 +1,68 @@
+import { sleep, group } from 'k6'
+import http from 'k6/http'
+import { SharedArray } from 'k6/data';
+import { htmlReport } from "https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js";
+import { textSummary } from "https://jslib.k6.io/k6-summary/0.0.1/index.js";
+
+const config = JSON.parse(open('./appConfig.json'));
+
+
+export const options = {
+  scenarios: {
+    DownloadNMFiles: {
+          exec: 'DownloadNMDailyFiles',
```

Comment:
I don't think this script calls either of the NMs weekly (https://msi-qa.admiralty.co.uk/NoticestoMariners) or daily (https://msi-qa.admiralty.co.uk/NoticestoMariners/ShowDailyFiles )pages.  It's really important that these are called so that the query against the FSS is triggered.

Reply:
Resolved